### PR TITLE
ENG-3864 Run all micro frontends

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: src/commands/run.ts [hunk 1]
```diff
@@ -32,7 +39,12 @@ export default class Run extends BaseExecutionCommand {
 
     this.validateInputs(Object.keys(flags).length, args.name)
 
-    if (flags['all-ms']) {
+    if (flags['all-mfe']) {
+      CliUx.ux.action.start(
+        `Running all micro frontends. Press ctrl + c to exit.`
+      )
+      await this.runAllComponents(Phase.Run, ComponentType.MICROFRONTEND)
+    } else if (flags['all-ms']) {
```
Comment (raxkaynan) [inline src/commands/run.ts:49]: If both `flags['all-mfe']` and `flags['all-ms']` are set, shouldn't it throw a specific error? If so, the `exclusive` flag property can be used for it.

**Review by raxkaynan — CHANGES_REQUESTED** [review]

File: src/commands/run.ts (hunk 1 above)
Comment (alepintus) [reply src/commands/run.ts:49]: @raxkaynan thanks for the suggestion
@cecchisandrone WDYT? using the exclusive flag property the error message would be something like this:
`--all-mfe= cannot also be provided when using --all-ms= `

Comment (cecchisandrone) [reply src/commands/run.ts:49]: yes seems reasonable

Comment (alepintus) [reply src/commands/run.ts:49]: done

**Review by raxkaynan — APPROVED** [review]

File: src/commands/run.ts [hunk 2]
```diff
@@ -89,7 +103,10 @@ export default class Run extends BaseExecutionCommand {
       component => new ColorizedWritable(component.name, maxPrefixLength)
     )
 
-    const executorService = new ParallelProcessExecutorService(executionOptions)
+    const executorService = new ParallelProcessExecutorService(
+      executionOptions,
+      10
```
Comment (zonia3000) [inline src/commands/run.ts:108]: Didn't we say to put this equal to the number of components?

Comment (alepintus) [reply src/commands/run.ts:108]: @zonia3000 I'll commit that change in the next PR (--all)

**Pull request merged**